StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/DebugMessageHandler.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 // Emitted when the SDK should print to the console in debug mode.
 class DebugMessageHandler: ScriptMessageHandler<String> {
-    init(didReceiveMessage: @escaping (String) -> Void = { Swift.debugPrint($0) }) {
```

Comment:
Use our Log helper so it prepends "[StripeConnect] " to the message when we receive a debug message from the web view